minutes/2024-12-16.html

```diff
@@ -0,0 +1,96 @@
+<!--#set var="title" value="" -->
+<!--#include virtual="../inc/header.html" --> 
+<h1><!--#echo encoding="none" var="title" --></h1>
+<p><strong>Venue</strong> : Google Meet</p>
+<p><strong>In attendance</strong>: - Mathew Blowers (mb2528, Chair) -
+Max Smith (ms2922, Junior Treasurer) - Bekalu Terefe-Zenebe (btt31,
+Secretary)</p>
```

Comment:
Just checking that this isn't meant to be a list?

(I think it's fine as it is if that's what you want, but it looks a bit like it might be intended as a Markdown list that didn't get converted into HTML)